pom.xml

```diff
@@ -314,17 +314,15 @@
         <version.io.rest-assured>3.0.6</version.io.rest-assured>
         <version.io.smallrye.open-api>2.1.1</version.io.smallrye.open-api>
         <version.io.smallrye.opentracing>2.0.0-RC1</version.io.smallrye.opentracing>
-        <version.io.smallrye.reactive-utils>2.0.0</version.io.smallrye.reactive-utils>
+        <version.io.smallrye.reactive-utils>2.1.1</version.io.smallrye.reactive-utils>
         <version.io.smallrye.smallrye-common>1.5.0</version.io.smallrye.smallrye-common>
         <version.io.smallrye.smallrye-config>2.0.2</version.io.smallrye.smallrye-config>
         <version.io.smallrye.smallrye-fault-tolerance>5.0.0</version.io.smallrye.smallrye-fault-tolerance>
         <version.io.smallrye.smallrye-health>3.0.0</version.io.smallrye.smallrye-health>
         <version.io.smallrye.smallrye-jwt>3.0.0</version.io.smallrye.smallrye-jwt>
         <version.io.smallrye.smallrye-metrics>3.0.1</version.io.smallrye.smallrye-metrics>
         <version.io.smallrye.smallrye-mutiny>0.13.0</version.io.smallrye.smallrye-mutiny>
```

Comment:
@rhusar Upgrade mutiny to 0.14.0 from 0.13.0 and it should be fine. It seems to be the version expected by https://github.com/smallrye/smallrye-reactive-utils/blob/2.1.1/pom.xml#L47

Reply:
Done.